src/app/application.cpp

```diff
@@ -556,7 +557,15 @@ int Application::exec(const QStringList &params)
     }
 #endif // DISABLE_WEBUI
 #else
+    UIThemeManager::initInstance();
     m_window = new MainWindow;
+    auto changeStyleSheet = [&]()
+    {
+        setStyleSheet(""); // following setStyleSheet fails otherwise
+        setStyleSheet(UIThemeManager::instance()->styleSheet());
+    };
+    connect(UIThemeManager::instance(), &UIThemeManager::themeChanged, changeStyleSheet);
+    changeStyleSheet();             // initialize window with previously set style
```

Comment:
Can this be moved into `MainWindow` constructor?

Reply:
glassez said it better to change the stylesheet for the `main application`

Reply:
No, I mean move it into MainWindow and change qapplication stylesheet from there.
https://doc.qt.io/qt-5/qapplication.html#qApp

styleSheet operations doesn't belong to application class IMO.

Reply:
It can be also done in UIThemeManager class itself.

Reply:
done